### PR TITLE
#168049990 Setup dotenv

### PR DESCRIPTION
**Pull request by Jordybastien**
#### What does this PR do?
This PR contains dotenv configurations and fixing some errors in server file turned on by ESLint
#### Description of Task to be completed?
Create your own dotenv file from this ".env.example" file
#### How should this be manually tested?
N/A
#### Any background context you want to provide?
N/A
#### What are the relevant pivotal tracker stories?
[168049990](https://www.pivotaltracker.com/story/show/168049990)
#### What are the relevant Github Issues?
N/A
#### Screenshots (if appropriate)
N/A
#### Questions:
N/A

### CODE REVIEW

File: src/index.js [hunk 1]
```diff
@@ -1,93 +1,98 @@
...
+/* eslint-disable no-console */
+/* eslint-disable import/no-unresolved */
+/* eslint-disable no-unused-vars */
+import dotenv from 'dotenv';
+import fs from 'fs';
+import http from 'http';
+import path from 'path';
+import methods from 'methods';
+import express from 'express';
+import bodyParser from 'body-parser';
```
Comment (mucyomiller) [inline src/index.js:10]: except that you used es6 and you didn't provide babel config for ES6+ support, other is okay 👍

File: .env.example [hunk 2]
```diff
@@ -0,0 +1,15 @@
...
+DB_USER=
+DB_PASS=
+DB_PORT=
+DB_HOST=
+
+#server port
+PORT=
+
+#JWT Secret key
+JWT_SECRET=
```
Comment (gik-hub) [inline .env.example]: Quite in order, just add a new line at the end of the file

**Review by amilykassim — APPROVED** [review]

**Review by Paulvitalis200 — COMMENTED** [review]
Hey @Jordybastien,
1. We need to install the `dotenv` dependency from npm. I don't see it anywhere in the `package.json`
2. The branch has to be named correctly. Remove the hashtag. I think you will need to raise another Pull Request when you rename this branch.

**Comment by Jordybastien** [comment]
> Hey @Jordybastien,
> 
> 1. We need to install the `dotenv` dependency from npm. I don't see it anywhere in the `package.json`

@Paulstar200 It was already there when I first pulled the repository, so I just added on what was missing in the server file

**Comment by Paulvitalis200** [comment]
> > Hey @Jordybastien,
> > 
> > 1. We need to install the `dotenv` dependency from npm. I don't see it anywhere in the `package.json`
> 
> @Paulstar200 It was already there when I first pulled the repository, so I just added on what was missing in the server file

Good. You will, however, need to change the name of your branch. It does not follow the proper naming convention.

**Comment by Jordybastien** [comment]
> > > Hey @Jordybastien,
> > > 
> > > 1. We need to install the `dotenv` dependency from npm. I don't see it anywhere in the `package.json`
> > 
> > 
> > @Paulstar200 It was already there when I first pulled the repository, so I just added on what was missing in the server file
> 
> Good. You will, however, need to change the name of your branch. It does not follow the proper naming convention.

Yes I tried to do it and followed this [tutorial](https://multiplestates.wordpress.com/2015/02/05/rename-a-local-and-remote-branch-in-git/)
but what it does is it closes this PR and require me to open a new one, so I'm here asking if I should proceed that way or just keep the lesson and do it well on the next branches
Thank you

**Review by mucyomiller — APPROVED** [review]
LGTM

**Pull request closed without merging**